S-Smans.Calculator/S-Smans.Calculator/Program.cs

```diff
@@ -0,0 +1,75 @@
+﻿using CalculatorLibrary;
+
+bool endApp = false;
+Calculator calculator = new Calculator();
+
+
+while (!endApp)
+{
+    Console.Clear();
+    Console.WriteLine("Console Calculator in C#\r");
+    Console.WriteLine("------------------------");
+    Console.WriteLine($"    {calculator.Iterations} calculation\n");
```

Comment:
I think you could be more descriptive with this message. I couldn't initially figure out what "1 calculation meant". Plus when I start the app shouldn't I have 0 calculations?

Reply:
You are right. Renamed it to make more sense and initialized it with 0.